sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -44,33 +48,32 @@ export interface AttestationCertificateManagementBody {
 }
 
 // @public (undocumented)
-export class AttestationClient extends AttestationClientContext {
-    constructor(credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials, instanceUrl: string, options?: AttestationClientOptionalParams);
+export class AttestationClient {
+    constructor(credentials: TokenCredential, instanceUrl: string, options?: AttestationClientOptions);
     // (undocumented)
     attestation: Attestation;
+    // Warning: (ae-forgotten-export) The symbol "GeneratedClient" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    BaseClient(): GeneratedClient;
```

Comment:
TS doesn't seem to have the construct of "Internal" - something that is public for the module but not exposed to the public.

However this is an interim construct intended to keep the SDK alive. It will be gone before the SDK is completed.

Reply:
In general, we hide those thins with `@hidden` tag but since this is going to be private before the release, you do not have to tag it. Actually I would recommend to keep to serve as a remainder that it needs to go private.

Reply:
DIdn't know about @hidden. Nice to know. If I get this right, it's not just hidden, it's gone for the release - this only exists because the existing operations directory classes need it.